code/modules/mob/living/living.dm

```diff
@@ -1805,3 +1802,13 @@ Thanks.
 	// TODO which is pretty irrelevant now but should be fixed
 	tool.reagents.reaction(src, INGEST)
 	return ..()
+
+/mob/living/proc/ApplySlip(var/obj/effect/overlay/puddle/P)
+	return on_foot()
```

Comment:
Could use a comment explaining that this returns 1 if the slip is possible (which is later checked by ``..()``)